src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ForAll.txt

```diff
@@ -109,3 +109,12 @@ Error({Kind:ErrorKind.Numeric})
 
 >> ForAll(ParseJSON("[""1"",""a""]"), If(Text(ThisRecord) = "1", Value(ThisRecord) * 2, Value(ThisRecord)))
 Error({Kind:ErrorKind.InvalidArgument})
+
+>> ForAll(Table(Blank()),ThisRecord)
+Table(Blank())
+
+>> ForAll(Table(Blank()),Coalesce(ThisRecord))
+Table(Blank())
+
+>> ForAll([1,2,3], If(Value <> 2, {a:Value}))
+Table({a:1},{a:Blank()},{a:3})
```

Comment:
This is wrong - the expected result should be `Table({a:1},Blank(),{a:3})`. The type of the second argument in ForAll is a record, so its result should not be wrapped.